common/src/chain/config.rs

```diff
@@ -63,9 +66,14 @@ impl ChainConfig {
     pub const fn get_proof_of_work_config(&self) -> PoWChainConfig {
         PoWChainConfig::new(self.chain_type)
     }
+
+    pub const fn get_blockreward_maturity(&self) -> &BlockDistance {
+        &self.blockreward_maturity
+    }
 }
 
 const MAINNET_ADDRESS_PREFIX: &str = "mlt";
+const MAINNET_BLOCKREWARD_MATURITY: BlockDistance = BlockDistance::new(500);
```

Comment:
Is there a specific reason behind 500 here? btc is 100 right?

Reply:
Yes. If block time is 2 minutes (which is my goal eventually), then 500 is equivalent to 100 in bitcoin's 10 minutes.

Reply:
> block time is 2 minutes
When did this change from 1 min block times?

Reply:
It never really changed. It's my esoteric plan that I mention every now and then because I think 1 minute is a bad idea. There will be a discussion about it some time later since we're now busy, and we'll decide what to put in these numbers, so I didn't bring this up.

Reply:
> This should probably also go into chainconfig eventually

It's there already